iotdb-core/consensus/src/main/java/org/apache/iotdb/consensus/pipe/PipeConsensusServerImpl.java

```diff
@@ -510,6 +510,24 @@ public synchronized void dropConsensusPipeToTargetPeer(Peer targetPeer)
     }
   }
 
+  public void startOtherConsensusPipesToTargetPeer(Peer targetPeer)
+      throws ConsensusGroupModifyPeerException {
+    final List<Peer> otherPeers = peerManager.getOtherPeers(thisNode);
+    for (Peer peer : otherPeers) {
+      if (peer.equals(targetPeer)) {
+        continue;
+      }
+      try {
+        consensusPipeManager.updateConsensusPipe(
```

Comment:
If a switchover leader is generated during the addpeer phase, the pipe state of the new leader has not been started at this time, is there some corner case that needs to be considered？

Reply:
no, only delay leader replicate without mistakes

Reply:
OK